### PR TITLE
Initiliazing for all FrameConexts for returnFocus api

### PR DESCRIPTION
**Pull request by Shyam-Gupta2011994**
Adding meetingStage FrameContext in returnFocus api

### CODE REVIEW

File: src/public/navigation.ts [hunk 1]
```diff
@@ -13,7 +13,10 @@ import { sendMessageToParent } from '../internal/communication';
  * @param navigateForward Determines the direction to focus in teams app.
  */
 export function returnFocus(navigateForward?: boolean): void {
-  ensureInitialized(FrameContexts.content);
+  ensureInitialized(
```
Comment (ydogandjiev) [inline src/public/navigation.ts:16]: Please change this to `ensureInitialized();`  since we will eventually want this API to work in all frame contexts.

**Review by ydogandjiev — CHANGES_REQUESTED** [review]

File: src/public/navigation.ts (hunk 1 above)
Comment (Shyam-Gupta2011994) [reply src/public/navigation.ts]: Done

**Review by ydogandjiev — APPROVED** [review]

**Review by cloudtx — APPROVED** [review]

File: src/public/navigation.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ import { sendMessageToParent } from '../internal/communication';
  * @param navigateForward Determines the direction to focus in teams app.
  */
 export function returnFocus(navigateForward?: boolean): void {
-  ensureInitialized(FrameContexts.content);
+  ensureInitialized();
```
Comment (ronakbakshi) [inline src/public/navigation.ts:16]: Are you trying to add meeting stage context only? Or do you want this API to work in all contexts?

Comment (Shyam-Gupta2011994) [reply src/public/navigation.ts:16]: Doing it for all now as per comments from Yuri

**Pull request merged**